tox.ini

```
@@ -72,3 +72,7 @@ deps = django>=2.1,<2.1.99
 [testenv:py3.7-d2.2]
 basepython = python3.7
 deps = django>=2.2,<2.2.99
+
+[testenv:py3.9-d3.2]
+basepython = python3.9
+deps = django>=3.0,<3.2.13
```

To stay in line with the other configs, this should probably say:

```suggestion
deps = django>=3.0,<3.2.99
```

Also don't forget about the new line.